openmc/mgxs/library.py

```diff
@@ -245,7 +247,8 @@ def by_nuclide(self, by_nuclide):
 
     @domain_type.setter
     def domain_type(self, domain_type):
-        cv.check_value('domain type', domain_type, tuple(openmc.mgxs.DOMAIN_TYPES))
+        cv.check_value('domain type', domain_type,
+                       tuple(openmc.mgxs.DOMAIN_TYPES))
```

Comment:
@wbinventor any reason you need the tuple line in there?  Shouldn't a list (which DOMAIN_TYPES already is) work too?